enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,256 @@
+
+```yaml
+apiVersion: config.openshift.io/v1alpha1
+kind: Backup
+metadata:
+  name: default
+spec:
+  etcd:
+    schedule: "0 0 * * *"
+    timeZone: "UTC"
+```
+
+
+
+
+### Risks and Mitigations
+
+## Test Plan
+
+An e2e test will be added to practice the scenario as follows 
+
+- Enable AutomatedBackupNoConfig FeatureGate.
+- Verify that a `Etcdbackup` CR has been installed.
+- Verify that a backup has been taken.
+- Verify that the backups are valid according to the retention policy.
+
+## Graduation Criteria
+TBD
+
+### Dev Preview -> Tech Preview
+TBD
+
+### Tech Preview -> GA
+TBD
+
+### Removing a deprecated feature
+TBD
+
+## Upgrade / Downgrade Strategy
+TBD
+
+## Version Skew Strategy
+TBD
+
+## Operational Aspects of API Extensions
+TBD
+
+#### Failure Modes
+TBD
+
+## Support Procedures
+TBD
+
+## Implementation History
+TBD
+
+### Drawbacks
+
+## Alternatives
+
+**Ideally**, the storage solution to use would have the following characteristics
+- Backups should be taken on a round-robin fashion on each master node to avoid overwhelming a specific etcd member more than others.
+- Spreading the backups among all etcd cluster members provide guarantees for disaster recovery in case of losing two members at the same time.
+- One fits all storage solution, to be used across all Openshift variants including Bare metal and Single Node Openshift.
+
+Below is each storage solution, with its pros and cons.
+
+### Dynamic provisioning using CSI
+
+As Openshift cluster that are based on cloud infrastructure offers dynamic storage provisioning capabilities based on CSI, allocating persistent storage for the etcd backups is trivial.
+
+Depending on the storage capabilities offered by the cloud provider, the allocated persistent volume could be attached to any of the master nodes and the backups can be used by any etcd member on any of the master nodes for restoration.
+
+* Pros
+  - Utilising CSI storage is ideal as it is well tested and allocating storage required defining a `PVC` manifest only.
+  - Some storage solutions based on CSI allows a `PV` to be attached to multiple nodes. This is ideal for both taking backup and restoration from various master nodes, in case one of the nodes is unreachable.
+  - It allows recovery even on disaster scenarios, where no master node is accessible, since the storage is allocated on the provider side.
+* Cons
+  - CSI is not an option for OCP `BM` or `SNO`.
+  - Some cloud providers allocate storage to be accessible only within the same availability zone. Since Openshift cluster allocate each master node on a different zone for high availability purposes, the backup will not be accessible if the master node where the backup was taken is no longer accessible.
+
+### Local Storage
+
+Relying on local storage works for all Openshift variants (i.e. cloud based, SNO and BM). There are three possible approaches using local storage, each is detailed below.
```

Comment:
On SNO, this doesn't seem particularly helpful. What is the recommendation going to be for SNO users?

Reply:
SNO users can create a PV with `hostPath` and a PVC pointing to it. Also, most SNOs have LVM-backed provisioner so its not a major problem

Reply:
so `hostPath` works well for it 👍🏽